### PR TITLE
docs: add parceljs browser example

### PR DESCRIPTION
**Pull request by raoulmillais**
This is an example of developing and building a Dapp which connects to IPFS that is bundled by Parcel.js: one of the desired examples in https://github.com/ipfs/js-ipfs/issues/1436

A version served over IPFS can be found [here](https://ipfs.io/ipfs/QmddoNMrm4Hg1h6AAxNYvJW4am3HxHXKW3HtZNYc6X66aw/) (for the commit that opens this PR)

**Notes and known limitations**

* Tested and working on OSX Mojave with node v10.13.0
* https://github.com/ipfs/js-ipfs/issues/1509 is still an issue
* Uses the "repo local" js-ipfs so `add`ing and `cat`ing are using the newer root API calls.

Please be gentle this is my first non-typo IPFS contribution, I look forward to any feedback. :)

### CODE REVIEW

**Comment by raoulmillais** [comment]
I'm getting a bit lost trying to investigate the build failures.  The OSX tests are failing for a timeout.  Master seems to be built using CIrcleCI and the PR by jenkins, both seem to have been failing for a while now (going by red crosses next to all first page of commits on master and all first page of open pull requests). Should I be concerned about this?  If so, could someone give me a pointer please?

**Comment by raoulmillais** [comment]
While playing around with IPFS using Parcel.js I came across a problem with babel-preset-env transpiling to aggressively and breaking `for...of` loops. See [this issue](https://github.com/isomorphic-git/isomorphic-git/issues/588) for more details.  Reconfiguring browserslist to only support latest 2 chrome versions prevents hitting this problem.

File: examples/browser-parceljs/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "playground",
```
Comment (alanshaw) [inline examples/browser-parceljs/package.json]: Shall we change to "browser-parceljs"?

**Comment by raoulmillais** [comment]
I've made the requested changes.  I haven't rebased the pull request off master and this PR is now behind, should I rebase @alanshaw?

**Comment by raoulmillais** [comment]
The commitlint check is failing because I accidentally missed the 's' off docs on [this commit](https://github.com/ipfs/js-ipfs/pull/1726/commits/b04cfcc6cf2019599efd446d58015875b50ec6ab) I'll have to rebase to fix this unless you plan to squash the commits when merging.  Sorry, I'm still getting to grips of the protocol you follow so I've been cautious and left it as-is.

**Comment by raoulmillais** [comment]
The force push was changing the README image to match the code on the tip of this PR

**Comment by raoulmillais** [comment]
Fastly is caching the README image, it will be in sync with the code when the cache expires

**Comment by alanshaw** [comment]
Changes look great. I'll give it a spin tomorrow morning and if it all works we can merge - thanks again!

**Pull request merged**